CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Views/Detail/SectionView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  SectionView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-11.
+//
+
+import SwiftUI
+
+struct SectionView: View {
+    var section: CourseSection = courseSections[0]
+    @EnvironmentObject var model: Model
+    @Environment(\.dismiss) var dismiss
+
+    var body: some View {
+        ZStack {
+            ScrollView {
+                cover
+                    .overlay(PlayView()
+                        .overlay(CircularView(value: section.progress, lineWidth: 5).padding(24)))
+                content
+                    .offset(y: 120)
+                    .padding(.bottom, 200)
+            }
+            .background(Color("Background"))
```

Comment:
@sghourab create please `enum AppColors` and put there all colours which you use in the app, instead of hardcoding them

Reply:
Hi Den, I've made the enum AppColors, but wanted to get your opinion on making an extension to Color instead because it would end up in more concise code: 

enum AppColors: String {
    case background = "Background"
    case shadow = "Shadow"
}
when called: 
Color(AppColors.background.rawValue)

vs 
extension Color {
 static let background = Color("Background")
}

when called: 
Color.background

Reply:
Never mind, thought about it a bit more and will use this approach: 
enum MyColor {
    case color(red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat)

    static let blue = Color(red: 0.29, green: 0.47, blue: 0.647)
    static let white = Color(red: 0.98, green: 0.98, blue: 1.0)
}